### PR TITLE
Update ko localization guide

### PR DESCRIPTION
**Pull request by seokho-son**
This PR updates the Korean localization guide. 
(https://kubernetes.io/ko/docs/contribute/localization_ko/)

This update reflects the discussion results in K8s Docs Korean L10n Team meetings regarding glossary localization.
- Localization of the API objects (`Kind`)
- Localization style of camelCase for the API objects
- Localization of the feature gates

### CODE REVIEW

**Comment by seokho-son** [comment]
cf) Preview page

https://deploy-preview-21572--kubernetes-io-master-staging.netlify.app/ko/docs/contribute/localization_ko/

**Comment by seokho-son** [comment]
PTAL @kubernetes/sig-docs-ko-reviews

File: content/ko/docs/contribute/localization_ko.md [hunk 1]
```diff
@@ -87,35 +87,83 @@ weight: 10
...
+
 {{% note %}}
 API 오브젝트의 필드 이름, 파일 이름, 경로와 같은 내용은 독자가 구성 파일이나
 커맨드라인에서 그대로 사용할 가능성이 높으므로 한글로 옮기지 않고 원문을 유지한다.
 단, 주석에 포함된 내용은 한글로 옮길 수 있다.
 {{% /note %}}
 
+### 기능 게이트(feature gate) 한글화 방침
+
+쿠버네티스의 [기능 게이트](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)를
```
Comment (sftim) [inline content/ko/docs/contribute/localization_ko.md:144]:
```suggestion
쿠버네티스의 [기능 게이트](/docs/reference/command-line-tools-reference/feature-gates/)를
```

File: content/ko/docs/contribute/localization_ko.md [hunk 2]
```diff
@@ -87,35 +87,83 @@ weight: 10
...
+
+기능 게이트의 예시는 다음과 같다.
+- Accelerators
+- AdvancedAuditing
+- AffinityInAnnotations
+- AllowExtTrafficLocalEndpoints
+- ...
+
+전체 기능 게이트 목록은
+[여기](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates)를 참고한다.
```
Comment (sftim) [inline content/ko/docs/contribute/localization_ko.md:155]:
```suggestion
[여기](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates)를 참고한다.
```

**Comment by seokho-son** [comment]
> Links within the documentation should be relative URIs. I've suggested changes with that in mind.

@sftim Thank you 👍  Accordingly, I updated this PR. (squashed commits into one commit)

**Comment by ysyukr** [comment]
@seokho-son 고생하셨습니다.
/lgtm

**Comment by gochist** [comment]
@seokho-son @ysyukr 감사합니다!
/approve

**Pull request merged**